.travis.yml

```diff
@@ -1,3 +1,5 @@
+sudo: false
```

Comment:
This is not required imo, not 100% sure.

Reply:
Then I think we should add it to our other repos :)